integrations/pagerduty/clients/pagerduty.py

```diff
@@ -0,0 +1,110 @@
+from typing import Any, Dict, List
```

Comment:
for dict and list you can use the built in list and dict keys

dict[str, Any] instead of Dict[str, Any]